_release-pypi/action.yml

```
@@ -43,6 +43,15 @@ inputs:
     required: true
     type: string
 
+  dry-run:
+    description: >
+      Whether to run or not this action in testing mode. Testing mode executes
+      all the action steps except the releasing artifacts one. This allows
+      developers to verify the health of the action once integrated in their
+      CI/CD pipelines without actually publishing artifacts.
+    required: true
```

I am wondering here if we should have the same arguments as for `release-github` action.
I was thinking that usually when you do a release, you want to publish on PyPI -so the logic should be similar-.
But it is actually not always true. So I am open to any decisions.

```suggestion
    required: false
    default: false
```

---

Dry-run should definitely be optional. This is must be optional.

---

Ahh now I see... This is the private action. In that case, that's fine

---

That's it, @RobPasMue. This is the private action. The convention is that private actions do not have optional arguments to be transparent when called from public actions.

By default, the "dry-run" is disabled in:

* `release-pypi-public`
* `release-pypi-private`
* `release-pypi-test`
* `release-github`

---

OK make sense. I did not notice that it was private. Thanks for the clarification.